elements/rh-chip/rh-chip.ts

```diff
@@ -0,0 +1,118 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { property } from 'lit/decorators/property.js';
+import { query } from 'lit/decorators/query.js';
+import { themable } from '@rhds/elements/lib/themable.js';
+import { consume } from '@lit/context';
+
+import { rhChipGroupSizeCtx } from './context.js';
+
+import '@rhds/elements/rh-icon/rh-icon.js';
+
+import styles from './rh-chip.css';
+
+export class ChipChangeEvent extends Event {
+  constructor(public checked: boolean) {
+    super('change', {
+      bubbles: true,
+      cancelable: true,
+    });
+  }
+}
+
+/**
+ * A chip is used to filter information or indicate that a selection was made.
+ * @summary Filter information or indicate that a selection was made
+ * @fires {ChipCheckedEvent} chip-checked - when chip is checked/unchecked
+ * @csspart chip - The main chip container
+ * @slot - The label of the checkbox
+ */
+@customElement('rh-chip')
+@themable
+export class RhChip extends LitElement {
+  static readonly styles = [styles];
+
+  /**
+   * Whether the chip is checked.
+   */
+  @property({
+    type: Boolean, reflect: true,
+    hasChanged(newVal: boolean, oldVal: boolean) {
+      return newVal !== oldVal;
+    },
+  })
+
+  set checked(value: boolean) {
+    const oldValue = this.#checked;
+    this.#checked = value;
+    if (this.chipInput) {
+      this.chipInput.checked = value;
+    }
+    this.requestUpdate('checked', oldValue);
+  }
+
+  get checked(): boolean {
+    return this.#checked;
+  }
```

Comment:
```suggestion
  @property({ type: Boolean, reflect: true }) checked?: boolean;
```

it's fine to just handle this in the template